libs/ledger/src/upow/work.cpp

```diff
@@ -102,7 +102,10 @@ Work::UInt256 Work::CreateHashedNonce() const
   hasher.Reset();
   hasher.Update(digest1);
 
-  return UInt256{hasher.Final()};
+  // Forcing little endian representation of the hash (even if it is actually
+  // represented in big endian encoding) in order to keep compatibility with
+  // proofs recorded in previously generated blocks.
+  return UInt256{hasher.Final(), true};
```

Comment:
@n-hutton is in the process of removing all the proof of work stuff. It doesn't make sense to worry about backwards compat.